content/xc9_k8s_aks.md

```diff
@@ -0,0 +1,108 @@
++++
+title = "XC9 + K8S + AKS = Bingo?! 🌟"
+date = "2019-11-06"
+tags = ["Docker", "Azure", "XC9"]
++++
+
+Sitecore 9 has a micro-service architecture and is designed with the cloud in mind. The community has picked this up by creating Docker images for Sitecore. This works great for local development, but it still has some challenges when deploying to the cloud. This changed with the recent introduction of Windows support for Kubernetes (K8S) and Azure Kubernetes Service (AKS) support for Windows containers. 
+
+<!--more-->
+This article introduces a Kubernetes setup, using Sitecore XC 9.1 as showcase, that allows you to deploy to AKS.
+
+# Sitecore XC 9.1 topology
+Let's first have a look at the Sitecore XC 9.1 topology. 
+The below diagram gives an overview of all the services and databases for a basic XC 9.1 setup.
+
+![](/sitecore_xc91_topology.png)
+
+These services are mapped to several Docker images, this mapping is as follows;
+
+- Content Management & Content Delivery are combined in a **Sitecore** Docker image
+- Business Tools and the Commerce Engine roles are grouped in a **Commerce** Docker image
+- The Identity Server has its own Docker image, named **Identity**
+- XConnect and its Processing part are placed in a **XConnect** Docker image
+- All SQL database are present in a **Mssql** image
+- All Solr cores are placed in a **Solr** image
+
+These 6 Docker images form the basis for running the containerized Sitecore XC system.
```

Comment:
Nog even iets zeggen over hoe je aan images komt? En wat de rol van een registry is?